### PR TITLE
[T4A5][T13-B2] Gabriel Gheorghian

### PR DESCRIPTION
**Pull request by GabrielGhe**
Ready for review.

### CODE REVIEW

**Comment by GabrielGhe** [comment]
Referenced #1250 for guidance

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+
+public class Tagging {
+    private static final String ADD_TAGGING = "+";
+    private static final String DELETE_TAGGING = "+";
+
```
Comment (KulaniT) [inline src/seedu/addressbook/data/tag/Tagging.java:9]: A good use of access modifiers :)

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+
```
Comment (KulaniT) [inline src/seedu/addressbook/data/tag/Tagging.java:5]: header comments

**Comment by KulaniT** [comment]
@GabrielGhe Overall nice work. 
 Some comments added. Please close the PR after reading comments

**Pull request closed without merging**